SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc

```diff
@@ -147,7 +147,15 @@ Plot {
 	}
 
 	domainCoordinates { |size|
-		var val = this.resampledDomainSpec.unmap(plotter.domain ?? { (0..size-1) });
+		var range, step, val, resamps;
+		range = domainSpec.range;
+		step =  if (range == 0) { 0 } { range / (size-1) };
```

Comment:
wouldn't `range / (size-1)` also return `0`?

Reply:
Yes, this could have been more explicit. The reason this check is here is that when plotting a single value, and range of the `ControlSpec` could be `0.0`. This breaks the arithmetic series used to generate `resamps`:

`(0, 0 + 0 .. 0)` works, while 
`(0, 0 + 0.0 .. 0)` breaks.

To be more explicit, the line should have been:
`step =  if (range == 0.0) { 0 } { range / (size-1) };`, or
`step =  if (range == 0.0) { range.asInt } { range / (size-1) };`

Importantly, this is a case that I found when plotting a function:

`p = { SinOsc.ar(440) }.plot(1);`

In the time between the plotter window opening and the server booting, calculating the output and loading it to an array to plot, the plotter's `value` is `[0.0]`, the default `domainSpec` is set to `[0,0].asSpec`, i.e. `range == 0.0`.

In any case, I've rewritten the method overall to be more explicit while handling a range of 0.0 as well as a data size of 1 by plotting the value in the middle of the plot. `Plotter` originally didn't plot a single value, which looked to me like a bug:

`[3].plot.plotMode_(\points)` >> window opens but nothing is plotted, and no grid/labels shown on X axis.